File: src/com/l2kt/gameserver/network/clientpackets/RequestDuelSurrender.java
package com.l2kt.gameserver.network.clientpackets;

import com.l2kt.gameserver.data.manager.DuelManager;

public final class RequestDuelSurrender extends L2GameClientPacket
{
	@Override
	protected void readImpl()
	{
	}
	
	@Override
	protected void runImpl()
	{
		DuelManager.getInstance().doSurrender(getClient().getActiveChar());
	}
}

File: src/com/l2kt/gameserver/network/clientpackets/RequestDuelSurrender.kt
package com.l2kt.gameserver.network.clientpackets

import com.l2kt.gameserver.data.manager.DuelManager

class RequestDuelSurrender : L2GameClientPacket() {
    override fun readImpl() {}

    override fun runImpl() {
        DuelManager.doSurrender(client.activeChar)
    }
}